src/components/refactor/rhf-inputs/text-input.js

```diff
@@ -22,6 +22,7 @@ import { isFieldRequired } from '../utils/utils';
 const TextInput = ({
     name,
     label,
+    labelValues,
```

Comment:
add a comment to explain its behaviour
I'm wondering if we should now allow "label" to be an object with all it's data instead, I don't know what solution would be the best